examples/notebooks/vanilla-gan-tutorial.ipynb

```diff
@@ -0,0 +1,1000 @@
+{
```

Comment:
<p>We are still waiting for <code>runner.infer</code>  to be implemented =)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/catalyst-team/catalyst/pull/660/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>